aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -607,34 +607,128 @@ SparseTensor& add_out_sparse_cpu(const SparseTensor& t, const SparseTensor& src,
 // add(Tensor, SparseTensor, Scalar)
 //    formerly known as spcadd
 // --------------------------------------------------------------------
-
 template <typename scalar_t>
-void add_dense_sparse_worker_cpu(Tensor& r, const Scalar& value, const SparseTensor& sparse, const Tensor& indices, const Tensor& values) {
+void add_dense_sparse_worker_non_hybrid_cpu(Tensor& r, const Scalar& value, const SparseTensor& sparse, const Tensor& indices, const Tensor& values) {
   auto indices_accessor = indices.accessor<int64_t, 2>();
   auto values_accessor = values.accessor<scalar_t, 1>();
 
   scalar_t* r_ptr = r.data_ptr<scalar_t>();
   auto r_strides = r.strides();
   scalar_t cast_value = value.to<scalar_t>();
   const auto sparse_dim = sparse.sparse_dim();
-
+  std::vector<int64_t> result_stride(sparse_dim);
+  for (int64_t d = 0; d < sparse_dim; d++) {
```

Comment:
Nit: please use `c10::irange`
```suggestion
  for (auto d = 0; c10::irange(sparse_dim)) {
```